src/test/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/DataflowReadsPipelineTest.java

```diff
@@ -24,6 +24,8 @@
 import java.util.Collections;
 import java.util.List;
 
+import static org.broadinstitute.hellbender.transformers.ReadTransformer.*;
```

Comment:
Wouldn't compile?  Or showed a red line? There's an intellij bug with their evaluation of this line but it should compile fine despite the warning.